src/main/components/TestComponent.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import DataTree from '../components/DataTree.jsx';
+
+const TestComponent = (props) => {
+  // data is the data response from msA 
+  // key is the unique ID for this test component-- maps to tests[i]
```

Comment:
I think using index as a key is going to break things sooner rather than later. Getting a warning for each test in React

> Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.

I think it’s okay for this PR but we should prioritize a fix 🔧